app/views/plate_picks/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= javascript_pack_tag 'plate_picks' %>
+<%= stylesheet_pack_tag 'plate_picks' %>
+
+<%= page_title 'Eloise\'s Plate Picks' %>
```

Comment:
Think rename that to something less personal.

Reply:
Was specifically requested in UAT